crates/sui-indexer-alt-graphql/src/api/types/transaction/mod.rs

```diff
@@ -253,18 +272,82 @@ impl Transaction {
     }
 }
 
+async fn tx_affected_address(
+    ctx: &Context<'_>,
+    Range {
+        start: tx_lo,
+        end: tx_hi,
+    }: Range<u64>,
+    page: &Page<CTransaction>,
+    SuiAddress(affected_address): SuiAddress,
+) -> Result<Vec<u64>, RpcError> {
+    let pg_reader: &PgReader = ctx.data()?;
+
+    let is_asc = page.is_from_front();
+    let query = query!(
+        r#"
+SELECT
+    tx_sequence_number
+FROM
+    tx_affected_addresses
+WHERE
+    {BigInt} <= tx_sequence_number /* tx_lo */
+    AND tx_sequence_number < {BigInt} /* tx_hi */
+    AND affected = {Bytea} /* affected_address */
+ORDER BY
+    CASE WHEN {Bool} /* is_asc */ = true THEN tx_sequence_number ELSE NULL END ASC,
+    CASE WHEN {Bool} /* is_asc */ = false THEN tx_sequence_number ELSE NULL END DESC
+LIMIT
+    {BigInt} /* limit */
```

Comment:
Once the sender filter is implemented, will we want to support sender + affected filters?

Reply:
Note that we only use `tx_senders` if the `sender` is the only filter applied. Otherwise we need to apply an additional filter on `sender` for whatever table we are currently querying (`sender` information is denormalized into every table, and we maintain an index that includes the sender on top of the primary index to support this query pattern).